packages/e2e/__tests__/servers/server.ts

```diff
@@ -0,0 +1,154 @@
+import * as mongoose from 'mongoose';
+import { ApolloServer, gql } from 'apollo-server';
+import { merge } from 'lodash';
+import fetch from 'node-fetch';
+
+// Server
+import { AccountsServer } from '@accounts/server';
+import { AccountsPassword } from '@accounts/password';
+// TODO rename to AccountsMongo ?
+import { Mongo } from '@accounts/mongo';
```

Comment:
This server instance is tied to mongo at the moment. If the database is parameterized then we could test all the database drivers using the same set of tests.

Reply:
done